.buildkite/nightly-benchmarks/scripts/launch-trt-server.sh

```diff
@@ -50,7 +36,15 @@ cd tensorrtllm_backend
 git checkout $trt_llm_version
 tensorrtllm_backend_dir=$(pwd)
 git submodule update --init --recursive
-cp -r ${tensorrt_demo_dir}/triton_model_repo ${tensorrtllm_backend_dir}/
+
+
+mkdir triton_model_repo
+cp -r all_models/inflight_batcher_llm/* triton_model_repo/
+python3 ../tools/fill_template.py -i tensorrt_llm/config.pbtxt triton_backend:tensorrtllm,engine_dir:OUTPUT,decoupled_mode:true,batching_strategy:inflight_fused_batching,batch_scheduler_policy:guaranteed_no_evict,exclude_input_in_output:true,triton_max_batch_size:2048,max_queue_delay_microseconds:0,max_beam_width:1,max_queue_size:2048,enable_kv_cache_reuse:false
+python3 ../tools/fill_template.py -i preprocessing/config.pbtxt triton_max_batch_size:2048,tokenizer_dir:$model_path,preprocessing_instance_count:5
```

Comment:
Regarding the configuration changes in tensorrt-demo, you can refer to my changes 3 weeks ago https://github.com/sgl-project/tensorrt-demo/commit/eb4abc6c42f86ac0a9a0e74fe679d044badb7890#diff-00ea7d3b5dcedcc01a61a048882ec23a595b80092849e728fb6a84daa6c792e7, the default configuration of https://github.com/neuralmagic/tensorrt-demo, such as instance count is basically wrong, and the default value of batch size is too small. It can be basically confirmed that if you test out worse performance of TensorRT LLM, it is likely due to incorrect configuration or testing errors. @KuntaiDu @alexm-neuralmagic

Reply:
Thank you for your reminder. I am adjusting these values. I have talked to Nvidia folks, and I will let them take a look at this PR to make sure the configurations look right to them.